tests/drivers/fprotect/app/testcase.yaml

```diff
@@ -0,0 +1,12 @@
+tests:
+  lib.fprotect.sys_init_fprotect:
+    platform_allow: nrf9160dk_nrf9160 nrf52840dk_nrf52840 nrf52dk_nrf52832
+      nrf5340dk_nrf5340_cpuapp
+    tags: fprotect
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "E: ***** BUS FAULT *****"
+        - "E:   Precise data bus error"
```

Comment:
```suggestion
```
See if it works without the harness config.

If not we should add more precise regex lines.